src/main/java/com/hyunec/cosmicbaseballinit/service/OneWeekGameService.java

```diff
@@ -0,0 +1,37 @@
+package com.hyunec.cosmicbaseballinit.service;
+
+import com.hyunec.cosmicbaseballinit.vo.GameResult;
+import com.hyunec.cosmicbaseballinit.vo.OneHitterResult;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OneWeekGameService implements GameService {
+
+    private Integer rounds;
+    private Integer myTeamScore;
+    private Integer otherTeamScore;
+
+    public OneWeekGameService() {
+        this.myTeamScore = 0;
+        this.otherTeamScore = 0;
+    }
+
+    public String gameStarted(){
+
+        return this.gameFinished(myTeamScore, otherTeamScore).getResultName();
+    }
+
+//    private OneHitterResult oneHitterGame(){
+//
+//
+//    }
+
+    public GameResult gameFinished(Integer myTeamScore, Integer otherTeamScore){
+        if(myTeamScore > otherTeamScore){
+            return GameResult.WIN;
+        }else if(myTeamScore < otherTeamScore){
```

Comment:
```suggestion
        } else if(myTeamScore < otherTeamScore){
```

컨벤션은 맞춤법과 같습니다. 조금 더 신경써주세요.

Reply:
네 알겠습니다. 고쳐야 하는 내용들은 저녁에 고쳐서 재리뷰 올리도록 하겠습니다. 리뷰 감사드립니다.